tap-setup-scripts/README_FIRST.md

```diff
@@ -0,0 +1,124 @@
+
+# Note: Run all commands from the top-level working directory
+
+## For Multi cluster
+
+update file user-input-values-mc.yaml with below params
+
+- the cluster name is a prefix of all 4 clusters
+- both build_cluster_arn & iterate_cluster_arn are required
+
+```
+cluster:
+  arch: multi
```

Comment:
nit - _arch_ is an overloaded term. would prefer something like _topology_ for the key here and the variable used elsewhere.

Reply:
sure, will do it in the next iteration.